src/System.Net.Security/src/System/Net/NTAuthentication.cs

```diff
@@ -486,10 +486,5 @@ private string GetClientSpecifiedSpn()
             }
             return spn;
         }
-
-        private int DecryptNtlm(byte[] payload, int offset, int count, out int newOffset, uint expectedSeqNumber)
```

Comment:
Two questions:

1. If this is removed, would the Negotiate protocol still work?
2. If yes, please switch the `NegotiateStreamPal.Windows.cs` definition of this function to be `private` (now it's `internal` to allow the removed call below).